es6/index.js

```diff
@@ -144,7 +144,9 @@ PrerenderSPAPlugin.prototype.apply = function (compiler) {
         PrerendererInstance.destroy()
         const msg = '[prerender-spa-plugin] Unable to prerender all routes!'
         console.error(msg)
+        console.log(err)
```

Comment:
Errors should be outputted to stderr, not stdout. This should be `console.error(err)`.

Reply:
Fixed. Thanks! Hope someone could review it and merge it... It has been two months...